src/components/ProjectSettingsTabs/AccessTab/getFormData.ts

```diff
@@ -0,0 +1,98 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import widgets from "components/SpruceForm/Widgets";
+import { radioBoxOptions } from "../utils";
+
+export const getFormData = (
```

Comment:
I wonder if theres a better name for this? Since `getTab` exports a variable called `formData` it was a little confusing trying to read it.  maybe `getFormSchema`? What do you think?